model-optimizer/extensions/ops/BatchNormInference.py

```diff
@@ -36,4 +37,6 @@ def __init__(self, graph: Graph, attrs: dict):
         }, attrs)
     @staticmethod
     def infer(node):
-        node.out_port(0).data.set_shape(node.in_port(0).data.get_shape())
+        output_shape = np.array(node.in_node(0).shape)
```

Comment:
Batch norm inference has just one output. So the change is invalid. You should use `copy_shape_infer` here instead of custom shape infer function.

Reply:
copy_shape_infer contains additional checks for value. Do we need this?